files/default/startManagerService.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+# The OOB start/stop node scripts don't explicitly set up the
+# profile name as a variable we can access; it is available
+# via the USER_INSTALL_ROOT property however.
+
+PROFILE_NAME=$( echo ${WAS_USER_SCRIPT} | awk -F '/' '{print $(NF-2)}')
```

Comment:
The comment mentions `USER_INSTALL_ROOT` but the script uses `WAS_USER_SCRIPT` - is that right?

Reply:
removed comment; there were two variables to choose from on the node agent but only one on the manage; consolidated on a single parameter now